apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/JobsTest.cs

```diff
@@ -67,5 +69,210 @@ public void ListJobs_FullProjection()
 
             Assert.NotNull(jobFound.Resource.Configuration);
         }
+
+        [Fact]
+        public void ListJobs_JobLabels()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+
+            var table = client.GetTable(_fixture.ProjectId, _fixture.DatasetId, _fixture.HighScoreTableId);
+            var jobToFind = client.CreateQueryJob($"SELECT * FROM {table}", null, new QueryOptions() { Labels = JobLabels});
+
+            // Find the job after listing with full projection.
+            var jobFound = (from job in client.ListJobs(new ListJobsOptions() { Projection = ProjectionEnum.Full })
```

Comment:
I'd write this without the query expression:

```csharp
var options = new ListJobsOptions { Projection = ProjectionEnum.Full };
var jobFound = client.ListJobs(options).Single(job => job.Reference.JobId == jobToFind.Reference.JobId);
```

Reply:
Done, definitely cleaner that way.